api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/calculation/pvls/RoutineCalculationTest.java

```diff
@@ -24,7 +26,7 @@ public PatientCalculation getCalculation() {
 
   @Override
   public Collection<Integer> getCohort() {
-    return Arrays.asList(new Integer[] {2, 6, 7, 8, 999, 432});
+    return Arrays.asList(new Integer[] {13, 6, 7, 8, 999, 432});
```

Comment:
Had to create a new scenario because after the fix criteria2 matches for patient 2 and this scenario is shared with other tests.